app/src/components/dialog/EditDialog.tsx

```diff
@@ -48,7 +48,7 @@ export interface IEditDialogProps {
    *
    * @memberof IEditDialogProps
    */
-  onSave: (values: any) => void;
+  onSave: (values: any, helper?: any) => void;
```

Comment:
question: why did we add the helper here?

Reply:
The helpers are used to help track the indices of the funding list, and this is data that is kept in the state before the initial post. We use the same mechanism for the iucn and permits.

Reply:
We also need to add the helpers to the onSave call below

```Typescript
onSubmit={(values, helpers) => {
  props.onSave(values, helpers);
}}>
```

Reply:
Can chat after, won't be a problem in either case

Reply:
turns out we don't need them ... removed now